### PR TITLE
[CTW-385] Add "Add Conditions" button and change text

### PR DESCRIPTION
**Pull request by WiktorKania**
Ticket: [CTW-385](https://zeushealth.atlassian.net/browse/CTW-385)

Screenshot 2022-10-19 at 15 31 58

### CODE REVIEW

File: src/components/core/table/table.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ export type TableProps<T extends MinRecordItem> = {
   columns: TableColumn<T>[];
   isLoading?: boolean;
   message?: string;
+  elements?: ReactElement;
```
Comment (WesleyKapow) [inline src/components/core/table/table.tsx:32]: This is a non descriptive name. How about changing `message` and `elements` to the single `emptyMessage` which can be a `string | ReactElement`. That way it is on the consumer to specify the entirety of the empty body.

File: src/components/content/conditions.tsx [hunk 2]
```diff
@@ -112,6 +114,18 @@ export function Conditions({ className, readOnly = false }: ConditionsProps) {
     );
   };
 
+  const addCondtiotnBtn = (
+    <p>
```
Comment (WesleyKapow) [inline src/components/content/conditions.tsx:118]: Why the `p`? Favor using parent div's with `space-y` (and maybe `flex`) if you want to create whitespace between elements.

**Review by WesleyKapow — DISMISSED** [review]
Minor nit about use of `p`, otherwise looks great!

File: src/components/content/conditions.tsx (hunk 2 above)
Comment (WiktorKania) [reply src/components/content/conditions.tsx]: `space-y` won't work for me for whatever reason, I used `my` instead

File: src/components/content/conditions.tsx [hunk 3]
```diff
@@ -33,7 +33,9 @@ export type ConditionsProps = {
   readOnly?: boolean;
 };
 
-const EMPTY_MESSAGE = "No conditions found";
+const EMPTY_MESSAGE_RECORD =
```
Comment (luthfib) [inline src/components/content/conditions.tsx:36]: Maybe rename to `EMPTY_MESSAGE_PATIENT_RECORD`? With `EMPTY_MESSAGE_RECORD` I thought it was being used in multiple places as I thought it was generic.

**Review by luthfib — COMMENTED** [review]
Looks good, left minor comment.

**Comment by luthfib** [comment]
Also don't forget to add changeset. Instructions on adding a changeset can be found https://github.com/zeus-health/ctw-component-library/blob/main/docs/adding-a-changeset.md

**Review by luthfib — APPROVED** [review]
Once you add a changeset, LGTM

File: .changeset/ten-dingos-jump.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": patch
+---
+
+update null states for condition tables
```
Comment (luthfib) [inline .changeset/ten-dingos-jump.md:5]: nit: `Update null states for condition tables.`

**Review by luthfib — APPROVED** [review]

File: src/components/content/conditions.tsx [hunk 5]
```diff
@@ -112,6 +114,16 @@ export function Conditions({ className, readOnly = false }: ConditionsProps) {
     );
   };
 
+  const addCondtiotnBtn = (
```
Comment (pmadria3) [inline src/components/content/conditions.tsx:117]: misspelled but will address in my PR

**Pull request merged**